src/Components/Aspire.Azure.AI.OpenAI/README.md

```diff
@@ -121,16 +121,16 @@ dotnet add package Aspire.Hosting.Azure
 Then, in the _Program.cs_ file of `AppHost`, add an Azure AI OpenAI service and consume the connection using the following methods:
```

Comment:
@mitchdenny - it looks like all the Azure Components READMEs are now broken since they just say

````
In your AppHost project, install the Aspire Azure Hosting library with [NuGet](https://www.nuget.org):
```dotnetcli
dotnet add package Aspire.Hosting.Azure
```
````

These need to be updated now that they are split into different packages.

Reply:
I can modify those in this PR as well.